bad_connector_woocommerce/models/stock_picking/exporter.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from odoo import _
+from odoo.exceptions import ValidationError
+from odoo.tools import html2plaintext
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import mapping
+from odoo.addons.connector.exception import MappingError
+
+_logger = logging.getLogger(__name__)
+
+
+class WooStockPickingRefundExporterMapper(Component):
+    _name = "woo.stock.picking.refund.export.mapper"
+    _inherit = "woo.export.mapper"
+    _apply_on = "woo.stock.picking.refund"
+
+    @mapping
+    def quantity_and_amount(self, record):
+        """Mapping for Quantity and Amount"""
+        line_items = []
+        total_amount = 0.00
+        sale_order_products = {
+            order_line.product_id.id for order_line in record.sale_id.order_line
+        }
+        for move in record.move_ids:
+            if move.product_id.id in sale_order_products:
+                order_line = next(
+                    (
+                        order_line
+                        for order_line in record.sale_id.order_line
+                        if order_line.product_id.id == move.product_id.id
+                    ),
+                    None,
+                )
+                if not order_line:
+                    continue
+                quantity_done = move.quantity_done
+                price_unit = float(order_line.woo_bind_ids[0].price_unit)
```

Comment:
Can we take order_line.woo_bind_ids[0] into a new variable, seems that it is so much in use.